docs/eventing/samples/kafka/channel/README.md

```diff
@@ -0,0 +1,136 @@
+# Kafka CRD default channel
+
+With the Apache Kafka CRD based channel installed, you can configure Knative to use `KafkaChannels` as default channel implementations. 
+
+## Prerequisites
+
+Install the `KafkaChannel` component on your configured cluster:
+   ```
+   curl -L "https://github.com/knative/eventing-contrib/releases/download/v0.8.2/kafka-channel.yaml" \
+    | sed 's/REPLACE_WITH_CLUSTER_URL/my-cluster-kafka-bootstrap.my-kafka-namespace:9092/' \
+    | kubectl apply --filename -
+   ```
+
+> Note: The above assumes that you have Apache Kafka installed in the `my-kafka-namespace`.
```

Comment:
Ok, that assumes an installation of Kafka on `kafka` namespace,

following: https://strimzi.io/quickstarts/minikube/

HOWEVER, all a user really neeeds is a Kafka cluster that is accessible from the installtion of knative.

On k8s, with Strimzi, it's pretty darn easy, and the quickstart, it would be `kafka`,  @RichieEscarez

Reply:
I see, strimzi is required? I think thats where Im having trouble. We should add that to the prerequisites section and include instructions for those dependencies.

Reply:
OK, let me update it.

I can also script some install options, if that's OK, @RichieEscarez